.air.toml

```diff
@@ -0,0 +1,33 @@
+root = "."
+tmp_dir = "tmp"
+
+[build]
+  bin = "./tmp/main"
+  cmd = "go build -o ./tmp/main cmd/web/main.go"
+  delay = 1000
+  exclude_dir = ["assets", "tmp", "vendor"]
+  exclude_file = []
+  exclude_regex = []
+  exclude_unchanged = false
+  follow_symlink = false
+  full_bin = ""
+  include_dir = []
+  include_ext = ["go", "tpl", "tmpl", "html", ".env"]
```

Comment:
Should this include `gohtml`? That's what the templates currently use.

Reply:
@mikestefanello Atom and Sublime user? :D

Reply:
Goland for me.